### DISCUSSION TITLE
Add support for Rosestacker spawners

### DISCUSSION THREAD

**Discussion by FourNineRugz** [Zrips/CMI#7435] [Enhancement]
When I enable 

    # Prevent mob spawning from spawners
    Spawners: true

Spawners still work despite players being afk near them. I suppose this is because i'm using a third party spawner stacker plugin called Rosestacker. Suggesting support be enabled for spawner plugins like this and alike.

**Comment by CloudeLecaw** [comment]
the plugins may should try to work with cmi instead of cmi should try to add support for x of the xxx spawner plugins out there...
have you asked them allready for support cmis spawner mangement?

**Comment by hermanhesse** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Reply by CloudeLecaw** [reply]
This is a vacation auto-reply email from QQ mailbox. Hi, I am currently on vacation and cannot reply to your emails personally. I will get back to you as soon as possible after the holiday.

spam?
